### PR TITLE
cmds/core/cp:

### PR DESCRIPTION
**Pull request by ChriMarMe**
* Rename cpArgs function to run and adapt std pattern for run function
* Adapt existing tests
* Add more tests in existing test pattern.

Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

**Comment by ChriMarMe** [comment]
T_T always this codecov patch stuff.

**Comment by ChriMarMe** [comment]
WHOOP WHOOP!!!!!!!!

File: cmds/core/cp/cp.go [hunk 1]
```diff
@@ -69,66 +71,74 @@ func promptOverwrite(dst string) (bool, error) {
...
+
+func setupPostCallback(verbose bool, w io.Writer) func(src, dst string) {
+	return func(src, dst string) {
+		if verbose {
+			fmt.Fprintf(w, "%q -> %q\n", src, dst)
+		}
+	}
+}
+
+// run is a function whose eval the args
```
Comment (MDr164) [inline cmds/core/cp/cp.go:118]: nit: Is "whose eval the args" really correct english? It sounds weird to me

File: cmds/core/cp/cp_test.go [hunk 2]
```diff
@@ -77,25 +79,92 @@ func createFilesTree(root string, maxDepth, depth int) error {
...
 }
 
-// TestCpsSimple make a simple test for copy file-to-file
-// cmd-line equivalent: cp file file-copy
-func TestCpSimple(t *testing.T) {
-	tempDir := t.TempDir()
-
-	f, err := randomFile(tempDir, "src-")
+func TestRunSimple(t *testing.T) {
+	tmpDir := t.TempDir()
```
Comment (MDr164) [inline cmds/core/cp/cp_test.go:83]: Are those being removed again? There are more occurrences throughout this test file where I don't think the temp files are being cleared up.

Comment (ChriMarMe) [reply cmds/core/cp/cp_test.go:83]: > TempDir returns a temporary directory for the test to use. The directory is automatically removed by Cleanup when the test and all its subtests complete. Each subsequent call to t.TempDir returns a unique directory; if the directory creation fails, TempDir terminates the test by calling Fatal.

**Pull request closed without merging**